test_fms/fms2_io/test_fms2_io.sh

```diff
@@ -12,5 +12,7 @@
 rm -f *.nc > /dev/null 2>&1
 rm -f *.nc.* > /dev/null 2>&1
 rm -f logfile.*.out > /dev/null 2>&1
+rm -f input.nml > /dev/null 2>&1
+cp input_base.nml input.nml
```

Comment:
It's actually better to remove these files in CLEANFILES in Makefile.am. That way, after the test is run, the files are still there for debugging purposes. Only when the user wants to clean them do they get erased.